vllm/outputs.py

```diff
@@ -113,19 +114,30 @@ def __init__(
         self.encoder_prompt_token_ids = encoder_prompt_token_ids
 
     @classmethod
-    def from_seq_group(cls, seq_group: SequenceGroup) -> "RequestOutput":
-        if seq_group.sampling_params is None:
+    def from_seq_group(
+        cls,
+        seq_group: SequenceGroup,
+        prior_output_lens: Dict[int, Tuple[int, int]],
+        include_prompt: bool = True,
```

Comment:
Can this parameter be inferred by checking the output kind and prior output lens? We only don't want the prompt if it's a delta response and the prior output lens are non-zero, right?

Reply:
@joerunde that's effectively what's already done but yes good idea, can encapsulate it within the method

Reply:
Thanks @joerunde that was a nice simplification